core/spec/models/spree/line_item_spec.rb

```diff
@@ -13,17 +13,19 @@
     end
   end
 
-  context '#destroy' do
-    it "fetches deleted products" do
-      line_item.product.destroy
+  context '#discontinued' do
+    it "fetches discontinued products" do
+      line_item.product.discontinue!
       expect(line_item.reload.product).to be_a Spree::Product
     end
 
-    it "fetches deleted variants" do
-      line_item.variant.destroy
+    it "fetches discontinued variants" do
+      line_item.variant.discontinue!
       expect(line_item.reload.variant).to be_a Spree::Variant
     end
+  end
 
+  context '#destroy' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.